src/AspNetCore/Inc/outprocessapplication.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#pragma once
+
+#include "application.h"
+
+class OUTPROCESS_APPLICATION : public APPLICATION
+{
+public:
+    OUTPROCESS_APPLICATION();
+
+    ~OUTPROCESS_APPLICATION();
+
+    __override
+    HRESULT Initialize(_In_ APPLICATION_MANAGER* pApplicationManager,
+                       _In_ ASPNETCORE_CONFIG*   pConfiguration);
+
+    __override
+    VOID OnAppOfflineHandleChange();
+
+    __override
+   REQUEST_NOTIFICATION_STATUS
+   ExecuteRequest(
+       _In_ IHttpContext* pHttpContext
+   );
+
+    HRESULT
+    GetProcess(
+        _In_    IHttpContext          *context,
+//        _In_    ASPNETCORE_CONFIG     *pConfig,
+        _Out_   SERVER_PROCESS       **ppServerProcess
+    )
+    {
+        return m_pProcessManager->GetProcess(context, m_pConfiguration, ppServerProcess);
+    }
+
+private:
+ /*   HRESULT
```

Comment:
I planed to do the wrap request processing logic into ExecuteRequest. This will be a big change. I did some prototype and won't check it in now. This is why it got commented out

Reply:
remove them for now for clean check-in